src/main/java/Application.java

```diff
@@ -54,33 +52,35 @@ public static void main(String[] args) {
 
         //Car service
         Car fiatCar = new Car("500", fiatMaker);
-        Car opelCar = new Car("Blitz", fiatMaker);
-        Car tavriaCar = new Car("Slavuta", tavriaMaker);
-        Car ladaCar = new Car("Granta", ladaMaker);
-        final Manufacturer fordMaker = new Manufacturer("Ford", "USA");
+        Car opelCar = new Car("Blitz", opelMaker);
+        final Car tavriaCar = new Car("Slavuta", tavriaMaker);
+        final Car ladaCar = new Car("Granta", ladaMaker);
```

Comment:
let's declare the variables closer to their first usage

Reply:
Done